### DISCUSSION TITLE
Invalid arguments

### DISCUSSION THREAD

**Discussion by akeelm** [technology-ebay-de/react-prebid#136] [General]
Keep seeing this error and the ads refresh: 

pubads_impl.js?cb=31082082:10 [GPT] Invalid arguments: PubAdsService.refresh([[object Object]]).

The config looks good and the ads do display

**Comment by caffeinated-pixels** [comment]
Funnily enough, I tried debugging this myself yesterday but didn't get far.  Which version of React Advertising are you using and are any of your ad slots conditionally rendered?

**Reply by akeelm** [reply]
I'm on the latest version 4.2.11. 

The slot does conditionally render, but that includes the provider wrapped in that condition too - so not sure how relevant that is?

**Reply by caffeinated-pixels** [reply]
When I was debugging it yesterday, there seemed to be some correlation with conditionally rendered ad slots, but it could also just be a coincidence.

**Reply by akeelm** [reply]
I just removed the conditions / conditional rendering around it and it's still showing that warning. So not sure if it's related.

**Reply by caffeinated-pixels** [reply]
Probably not then, but thanks for testing that. I'll continue looking into it; hopefully, I can figure out what's going wrong.

**Reply by akeelm** [reply]
Thanks! Hope you can figure it out!
